packages/integration-tests/projects/suite-web/tests/suite/bug-report-form.test.ts

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+
+
+// @group:suite
+// @retry=2
+
+describe('Stories of bug report forms', function () {
```

Comment:
This just work fine but for consistency with the rest of the code-base it would be better to use arrow functions 

So:
```javascript
describe('Stories of bug report forms', () => {
```

Reply:
I finally installed ESlint and seen the red sea done without the arrow :D